### DISCUSSION TITLE
Possibility to run the gpu at full power in standard mode (G15 2022)

### DISCUSSION THREAD

**Discussion by Phatixberg** [seerge/g-helper#194] [General]
Hi, Is it possible to run the gpu at full power (120w) in standard mode? Because there is no gpu related option in the Fans+Power section. Is this a Bug or not supported on g15? 
<img width="548" alt="Ekran görüntüsü 2023-04-05 071619" src="https://user-images.githubusercontent.com/128205131/229980173-f917ecc8-0f08-477a-9cff-fc821ae0bdea.png">

**Comment by seerge** [comment]
Hello, I think G15 supports only CPU power limits. Does it have power (in watts?) slider in armoury ?

**Reply by Phatixberg** [reply]
There were 3 options. 
<img width="316" alt="Ekran görüntüsü 2023-04-05 141426" src="https://user-images.githubusercontent.com/128205131/230064778-3f146503-461d-49db-b5c4-3798314fbdbe.png">

**Reply by seerge** [reply]
Well, it's not power. GPU clocks are controlled by nvidia driver, i don't have nvidia so can't add it safely in the app. This was discussed explicitly in https://github.com/seerge/g-helper/issues/75 

You can do same overclocking using MSI afterburner (as lots of people seem to do)

**Reply by Phatixberg** [reply]
Oh.. thank you again. I have one more question. Is smart display service required? I think it's preventing me from switching to eco mode. Whenever I switch to eco mode or unplugged on optimized mode the dgpu does not turn off. G-helper says I am in igpu mode but dgpu is still active. I can see it from task manager and g-helper still shows gpu temp.

**Reply by seerge** [reply]
It's not needed. How do you check that dGPU is active ? is it visible in device manager? 
As for temps - that's ok, app now fall backs to asus own mb sensor, that also work when dGPU is off :)

**Reply by Phatixberg** [reply]
hello again, Yes, is it visible in device manager and task manager. About temps... the laptop rarely manages to go into eco mode and then dgpu is actually disabled. when this happens the gpu temperature disappears for me. So something is going wrong in this case? my G-helper version is 0.46. I realized now, I really managed to switch to eco mode for now.
<img width="809" alt="Ekran görüntüsü 2023-04-06 015024" src="https://user-images.githubusercontent.com/128205131/230229600-9cf4f627-9300-4d80-9ea1-1a8d5c4a06b9.png">
 Currently dgpu is not showing up in task manager and device manager.

**Reply by seerge** [reply]
Well, I don't have extrasensory skills to check what happens if it doesn't switch (as you say), cause on screenshots it's obviously switched :) 

Try to find a pattern if it happens, mb you have external display connected or so, and bios refuses to switch. If you will notice some pattern - feel free to post logs from ``%AppData%\GHelper``
